requirements/common.txt

```diff
@@ -208,6 +209,19 @@ slugid==1.0.7 --hash=sha256:6dab3c7eef0bb423fb54cb7752e0f466ddd0ee495b78b763be60
 taskcluster==1.3.4 \
     --hash=sha256:c0bbaa680aed4e502c370d24258c85e6ea7a891fd090ac6af66356e0e9a459fc
 
+# Dependency for Taskcluster
+aiohttp==2.2.5 \
```

Comment:
For these could you add `; python_version >= '3'`, since they aren't needed on Python 2.